framework/src/bcs/DGFunctionDiffusionDirichletBC.C

```diff
@@ -42,7 +42,7 @@ DGFunctionDiffusionDirichletBC::DGFunctionDiffusionDirichletBC(const InputParame
 Real
 DGFunctionDiffusionDirichletBC::computeQpResidual()
 {
-  const unsigned int elem_b_order = static_cast<unsigned int> (_var.getOrder());
+  const unsigned int elem_b_order = _var.order();
```

Comment:
My only argument is that if we do it here then we need to everywhere, such as
```
for (unsigned int i=0; i<_var.order(); ++i)
```
which seems painful and unnecessary.